public/css/module.less

```diff
@@ -1479,15 +1491,16 @@ input[type=submit].icon-button {
 
 }
 .breadcrumb li a {
```

Comment:
Generally for .breadcrumb li a. The last one should be in text-color, since it isn’t interactive. We be cool, if you find a way to achieve this.


This is how it should look.
![Screen Shot 2021-11-23 at 17 35 41](https://user-images.githubusercontent.com/6977434/143065521-9c060abf-4ef7-48e4-a541-269fef17941a.png)

Reply:
For me, these breadcrumb list are all clickable till the end. But I changed it the way you requested.